src/components/HarvestersDialog/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import CardsGallery from '../CardsGallery'
+import Dialog from '../Dialog'
+
+export default React.memo(function HarvestersDialog(props) {
+  return (
+    <Dialog
+      id='harvesters-dialog'
```

Comment:
```suggestion
      id='harvesters-dialog'
      role='alertdialog'
```

This will require adding `role={props.role}` to the `Dialog` component as well:

```diff
return (
     <A11yDialog
+      role={props.role}
       id={props.id}
```

This makes it so you cannot close it by clicking the backdrop or pressing the ESC key and force you to interact with it. It can still be closed with the close button on the top right corner. I think it would be a good idea for that specific dialog, since there’s no way to undo it.

Reply:
That's what I wanted to do by removing the close attribute, perfect!